src/serac/physics/utilities/functional/functional.hpp

```diff
@@ -23,45 +24,52 @@
 #include "serac/numerics/assembled_sparse_matrix.hpp"
 #include "serac/infrastructure/logger.hpp"
 
-namespace serac {
+namespace detail {
 
-/**
- * @brief a type trait used to identify the Hcurl family
- */
-template <typename T>
-struct is_hcurl {
-  /**
-   * @brief Flag denoting the type contained is Hcurl
-   */
-  static constexpr bool value = false;
-};
+  struct elem_info{
```

Comment:
Needs doxygen

Reply:
Also, these methods are not capitalized by either our style guide (types in `PascalCase`, methods in `camelCase`) or MFEM's (types in `PascalCase`, methods in `PascalCase`).